Code/Framework/AzCore/AzCore/IO/IStreamer.h

```diff
@@ -235,6 +230,24 @@ namespace AZ::IO
         //! @return A reference to the provided request.
         virtual FileRequestPtr& FlushCaches(FileRequestPtr& request) = 0;
 
+        //! Retrieves statistics for the requested type. This is meant for statistics that can't be retrieved in a lockless manner. For
+        //! metrics that can be retrieved lockless use CollectStatistics.
+        //! @param output The storage for the information that's being reported. The container needs to remain alive for the duration of
+        //!     the call. Register a callback with SetRequestCompleteCallback to determine when the container is no longer needed.
+        //! @param reportType The type of information to report.
+        //! @return A smart pointer to the newly created request with the command to flush all caches.
```

Comment:
This says it will return a request to flush all caches, rather than report statistics.

Reply:
Fixed in the next commit.